alpha7 fails to build under bazel / cargo-raze rules

This appears to be due to a broken fallback - when the build script is skipped, the fallback code is currently broken, suggesting a testing hole as well.

```
INFO: Invocation ID: 4349d931-caf2-41bf-b502-1d84d7d00916
INFO: Analyzed target //gandalf-auth:gandalf-auth (9 packages loaded, 121 targets configured).
INFO: Found 1 target...
INFO: From Compiling Rust rlib value_bag v1.0.0-alpha.7 (16 files):
error[E0308]: mismatched types
  --> external/raze__value_bag__1_0_0_alpha_7/src/internal/cast/primitive.rs:11:57
   |
11 | pub(super) fn from_any<'v, T: 'static>(value: &'v T) -> Option<Primitive<'v>> {
   |               --------                                  ^^^^^^^^^^^^^^^^^^^^^ expected enum `Option`, found `()`
   |               |
   |               implicitly returns `()` as its body has no tail or `return` expression
   |
   = note:   expected enum `Option<Primitive<'v>>`
           found unit type `()`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0308`.
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "value-bag"
-version = "1.0.0-alpha.7"
+version = "1.0.0-alpha.8"
 authors = ["Ashley Mannix <ashleymannix@live.com.au>"]
 edition = "2018"
 license = "Apache-2.0 OR MIT"
@@ -46,7 +46,7 @@ error = [
 test = ["std"]
 
 [dependencies.sval1_lib]
-version = "1.0.0-alpha.5"
+version = "=1.0.0-alpha.5"
 optional = true
 default-features = false
 features = ["fmt"]
@@ -74,7 +74,7 @@ package = "erased-serde"
 version = "0.1"
 
 [dev-dependencies.sval1_lib]
-version = "1.0.0-alpha.5"
+version = "=1.0.0-alpha.5"
 features = ["test"]
 package = "sval"
 
@@ -83,7 +83,7 @@ version = "1"
 package = "serde_test"
 
 [dev-dependencies.sval1_json]
-version = "1.0.0-alpha.5"
+version = "=1.0.0-alpha.5"
 features = ["std"]
 package = "sval_json"
 

--- a/README.md
+++ b/README.md
@@ -10,20 +10,20 @@ Add the `value-bag` crate to your `Cargo.toml`:
 
 ```rust
 [dependencies.value-bag]
-version = "1.0.0-alpha.7"
+version = "1.0.0-alpha.8"
 ```
 
 You'll probably also want to add a feature for either `sval` (if you're in a no-std environment) or `serde` (if you need to integrate with other code that uses `serde`):
 
 ```rust
 [dependencies.value-bag]
-version = "1.0.0-alpha.7"
+version = "1.0.0-alpha.8"
 features = ["sval1"]
 ```
 
 ```rust
 [dependencies.value-bag]
-version = "1.0.0-alpha.7"
+version = "1.0.0-alpha.8"
 features = ["serde1"]
 ```
 

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -5,7 +5,7 @@
 //! without losing their original structure.
 
 #![cfg_attr(value_bag_capture_const_type_id, feature(const_type_id))]
-#![doc(html_root_url = "https://docs.rs/value-bag/1.0.0-alpha.7")]
+#![doc(html_root_url = "https://docs.rs/value-bag/1.0.0-alpha.8")]
 #![no_std]
 
 #[cfg(any(feature = "std", test))]

